dirs/dirs.go

```diff
@@ -240,7 +240,7 @@ func SetRootDir(rootdir string) {
 	GlobalRootDir = rootdir
 
 	isInsideBase, _ := isInsideBaseSnap()
-	if !isInsideBase && release.DistroLike("fedora", "arch", "archlinux", "manjaro", "antergos") {
+	if !isInsideBase && release.DistroLike("fedora", "arch", "archlinux", "manjaro", "manjaro-arm", "antergos") {
```

Comment:
The list grew a little. Can we have a variable that holds all the distros using `/var/lib/snapd/snap`?
```
var distrosWithVarLibSnapdSnap = []string{
	"fedora",
	"arch",
	"archlinux",
	"manjaro",
	// XXX: drop once Manjaro on ARM fixes their ID_LIKE (is there a bug report?)
	"manjaro-arm",
	"antergos",
}
if !isInsideBase && release.DistroLike(distrosWithVarLibSnapdSnap...) { 
	...
}
```

Reply:
As I said above, I do't believe manjaro-arm will change. It's their decision to name things.